source/mrmeshpy/MRPythonVoxels.cpp

```diff
@@ -63,6 +63,11 @@ MR_ADD_PYTHON_CUSTOM_DEF( mrmeshpy, Voxels, []( pybind11::module_& m )
         pybind11::arg( "simpleVolume" ), pybind11::arg( "cb" ) = MR::ProgressCallback{},
         "Make VdbVolume from SimpleVolume. Make copy of data.\n"
         "Grid can be used to make iso-surface later with gridToMesh function." );
+    m.def( "vdbVolumeToSimpleVolume",
+        MR::decorateExpected(
+            []( const MR::VdbVolume& volume, MR::ProgressCallback cb ) { return vdbVolumeToSimpleVolume( volume, {}, cb ); } ),
```

Comment:
no real need in lambda, just `MR::decorateExpected( &MR::vdbVolumeToSimpleVolume )` should work